PyDBtickets/TrainTicketDocument.py

```diff
@@ -111,47 +110,28 @@ def find_vat(self):
         For one ticket there might be charged full (19%) and discounted (7%) VAT.
         :return: float, vat
         """
-        def get_vat(rate):
-            pattern_rate = r"\nMwSt \(?D\)? {r}%\n(".format(r=rate) + self.amt_regex + r"\n)*"
-            # there is a price column stating the prices of all single positions and
-            # the total price in the end
-            # Hence it suffices to take the last amount
-            amts = re.search(pattern_rate.decode("utf-8"), self.ticket_text)
-            if amts:
-                last_amt = [x for x in amts.group().split("\n") if x != ""][-1]
-                vat_at_rate = float(re.search(r"[0-9,]+", last_amt).group().replace(",", "."))
-            else:
-                vat_at_rate = 0
-            return vat_at_rate
-
-        vat_19 = get_vat(int(self.vat_rate_full * 100))
-        vat_7 = get_vat(int(self.vat_rate_disc * 100))
+        def get_vat():
+            # todo: this does not support multiple positions yet
```

Comment:
if I recall correctly it was necessary to have multiple positions (either for return journeys or for tickets including a bicycle - not sure anymore)

Reply:
do you have any test data for this?

Reply:
I've sent you a bike ticket